File: plugin/src/org/jetbrains/yesod/hamlet/highlight/HamletColors.java
package org.jetbrains.yesod.hamlet.highlight;

/**
 * @author Leyla H
 */

import com.intellij.openapi.editor.colors.TextAttributesKey;

public interface HamletColors {
    TextAttributesKey OPERATOR = TextAttributesKey.createTextAttributesKey("HAMLET_OPERATOR");
    TextAttributesKey COMMENT = TextAttributesKey.createTextAttributesKey("HAMLET_COMMENT");
    TextAttributesKey ATTRIBUTE = TextAttributesKey.createTextAttributesKey("HAMLET_ATTRIBUTE");
    TextAttributesKey ATTRIBUTE_VALUE = TextAttributesKey.createTextAttributesKey("HAMLET_ATTRIBUTE_VALUE");
    TextAttributesKey STRING = TextAttributesKey.createTextAttributesKey("HAMLET_STRING");
    TextAttributesKey IDENTIFIER = TextAttributesKey.createTextAttributesKey("HAMLET_IDENTIFIER");
}

File: plugin/src/org/jetbrains/yesod/hamlet/highlight/HamletColors.kt
package org.jetbrains.yesod.hamlet.highlight

/**
 * @author Leyla H
 */

import com.intellij.openapi.editor.colors.TextAttributesKey

interface HamletColors {
    companion object {
        val OPERATOR: TextAttributesKey = TextAttributesKey.createTextAttributesKey("HAMLET_OPERATOR")
        val COMMENT: TextAttributesKey = TextAttributesKey.createTextAttributesKey("HAMLET_COMMENT")
        val ATTRIBUTE: TextAttributesKey = TextAttributesKey.createTextAttributesKey("HAMLET_ATTRIBUTE")
        val ATTRIBUTE_VALUE: TextAttributesKey = TextAttributesKey.createTextAttributesKey("HAMLET_ATTRIBUTE_VALUE")
        val STRING: TextAttributesKey = TextAttributesKey.createTextAttributesKey("HAMLET_STRING")
        val IDENTIFIER: TextAttributesKey = TextAttributesKey.createTextAttributesKey("HAMLET_IDENTIFIER")
    }
}
